src/lib/bootloader/stage1_proposal.rb

```diff
@@ -198,7 +199,9 @@ class PPC < Stage1Proposal
       def propose
         partition = proposed_prep_partition
         if partition
-          assign_bootloader_device([:custom, partition])
+          # ensure that stage1 device is in udev (bsc#1041692)
```

Comment:
No. "... is in udev **form**"
I am sick of careless language usage *that obscures the meaning which it is intended to illuminate*.